lib/test.js

```diff
@@ -1,20 +1,25 @@
 const fsPromises = require('fs').promises
-const fetch = require('node-fetch')
 const github = require('./github')
+const gitURLParse = require('git-url-parse')
 
 module.exports = async function (url) {
-  const dep = await getLocalPackageName()
-  console.log('Got local package name: ', dep)
-  const [org, repo] = getOrgRepo(url)
-  console.log(`Got org: ${org} and repo: ${repo}`)
-  const packageJSON = await github.getPackageJson(org, repo)
-  if (!checkPackageInPackageJSON(dep, packageJSON)) {
+  const parentPkgJSON = await getLocalPackageJSON()
+  const parentPkgInfo = gitURLParse(parentPkgJSON.repository.url)
+  console.log(`Parent module: ${parentPkgInfo.owner}/${parentPkgJSON.name}`)
+
+  const dependentPkgInfo = gitURLParse(url)
+  const dependentPkgJSON = await github.getPackageJson(dependentPkgInfo.owner, dependentPkgInfo.name)
+  console.log(`Dependent module: ${dependentPkgInfo.owner}/${dependentPkgInfo.name}`)
+
+  if (!checkPackageInPackageJSON(parentPkgJSON.name, dependentPkgJSON)) {
     throw new Error('Dependency not found in package.json')
```

Comment:
Not a part of this PR, but still:

```suggestion
    throw new Error(`${parentPkgInfo.owner}/${parentPkgJSON.name} not found in package.json of ${dependentPkgInfo.owner}/${dependentPkgInfo.name}`)
```